src/account/keycloak.service.ts

```diff
@@ -35,6 +36,22 @@ export class KeycloakService {
     );
   }
 
+  /**
+   * Delete a user in the realm with the given id
+   * @param realm
+   * @param id
+   */
+  deleteUser(realm: string, id: string) {
+    return this.perform(
+      this.http.delete,
+      `${realm}/users/${id}`,
+    ).pipe(
+      catchError(() => {
+        return of("")
```

Comment:
Does this mean we silently ignore any errors without logging or giving the caller a special return code? Should we really do that?

Reply:
We could ignore just 404 errors and forward all other codes to the user. The 404 case is some kind of expected, so ignoring it is fine for me.

Reply:
todo: will add response body with information if entity was deleted or not, add logs